### PR TITLE
fix: Fix dark mode small issues

### PR DESCRIPTION
**Pull request by mutdmour**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

**Review by alexgrozav — APPROVED** [review]

File: packages/design-system/src/css/_tokens.dark.scss
Comment (netroy) [inline packages/design-system/src/css/_tokens.dark.scss:40]: The contrast on this is not good.
image

Not sure `--color-secondary` needs to be brighter in dark mode, or if we need to define a new `--color-sticky-code-foreground` variable.

**Review by netroy — APPROVED** [review]
The sticky code is still difficult to read, but it's your call if you want to fix that in this PR, or if we should create another PR for that.
The rest looks good, thanks 👍🏽

File: packages/design-system/src/css/_tokens.dark.scss
Comment (mutdmour) [reply packages/design-system/src/css/_tokens.dark.scss:40]: ya it's not perfect.. but unblock this

**Comment by janober** [comment]
Got released with [`n8n@1.15.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.15.1)

**Pull request merged**
